docling_core/types/doc/document.py

```diff
@@ -458,8 +458,12 @@ def get_ref(self):
         populate_by_name=True,
     )
 
+    def path(self):
+        """Get the path of the reference."""
+        return self.cref.split("/")
```

Comment:
Instead of extending the public API, I would just add this as an internal helper function on the module level (e.g. `_split_ref_to_path(ref_str)`)

Reply:
done